train_configs/llama3_8b.toml

```diff
@@ -18,7 +18,7 @@ save_tb_folder = "tb"
 [model]
 name = "llama3"
 flavor = "8B"
-norm_type = "fused_rmsnorm"  # [layernorm / np_layernorm / rmsnorm / fused_rmsnorm]
+norm_type = "rmsnorm"  # [layernorm / np_layernorm / rmsnorm / fused_rmsnorm]
```

Comment:
Is this change intentional?

Reply:
yeah I feel it's a bit painful to switch between `rmsnorm` and `fused_rmsnorm` locally when switching between 1-D and 2-D sharding, so I changed `rmsnorm` to default, we can always default to `fused_rmsnorm` once @tianyu-l changes the fused_rmsnorm registration https://github.com/pytorch/torchtitan/pull/296.

Let me know if you think we should default to `fused_rmsnorm` for 8B

Reply:
Keeping it as `rmsnorm` sounds good to me!